.gitignore

```diff
@@ -5,3 +5,6 @@ bin/
 
 karydia*.pem
 cov.out
+
+manifests/
+manifests-dev/
```

Comment:
Why do we need `manifests-dev/` at the `.gitignore` file? Is this still used anywhere?

Reply:
You are right, someone forgot about that.